run.py

```diff
@@ -539,6 +591,24 @@ def do_gear_convert_stats(subject_id, dry_run, environ, metadata, log):
                 metadata["analysis"]["info"][f"{hh}_{pp}_stats_area_mm2"] = ap_json
 
 
+def do_gtmseg(subject_id, dry_run, environ, log):
```

Comment:
I am not sure of the rationale for not declaring the log at the top of the run.py file... and configuring the log with the gear toolkit.  Do they both point to the same log? Do they both give the same __name__?
What is the "best practices" going forward? 

Perhaps a broader conversation than this review comment.

Reply:
Good questions!  It would be great to decide as a group.